Doc/library/imaplib.rst

```diff
@@ -174,9 +174,9 @@ example of usage.
 
 .. seealso::
 
-   Documents describing the protocol, and sources and binaries  for servers
-   implementing it, can all be found at the University of Washington's *IMAP
-   Information Center* (https://www.washington.edu/imap/).
+   Source Code of Documents describing the protocol, sources and binaries for servers
```

Comment:
"Source code of documents" makes no sense.  Further, I don't think binaries can be found in the source repo.

Reply:
binaries are removed But what should i do of " Source Code of Documents describing"